src/signer_node/message_processor/process_blockvss.rs

```diff
@@ -31,103 +31,122 @@ where
     T: TapyrusApi,
     C: ConnectionManager,
 {
+    let new_shared_block_secrets = match store_received_vss(
+        sender_id,
+        prev_state,
+        vss_for_positive,
+        secret_share_for_positive,
+        vss_for_negative,
+        secret_share_for_negative,
+    ) {
+        Ok(shared_block_secrets) => shared_block_secrets,
+        Err(e) => {
+            error!("Error: {:?}, state: {:?}", e, prev_state);
+            return prev_state.clone();
+        }
+    };
+
+    // Returns if the vss count doesn't meet required number
+    if new_shared_block_secrets.len() != params.pubkey_list.len() {
+        match prev_state {
+            NodeState::Master { .. } => {
+                return Master::from_node_state(prev_state.clone())
+                    .shared_block_secrets(new_shared_block_secrets)
+                    .block_shared_keys(None)
+                    .build();
+            }
+            NodeState::Member { .. } => {
+                return Member::from_node_state(prev_state.clone())
+                    .shared_block_secrets(new_shared_block_secrets)
+                    .block_shared_keys(None)
+                    .build();
+            }
+            _ => {
+                return prev_state.clone();
+            }
+        }
+    }
+
+    let block_shared_keys = match broadcast_local_sig(
+        blockhash,
+        &new_shared_block_secrets,
+        priv_shared_keys,
+        prev_state,
+        conman,
+        params,
+    ) {
+        Ok((is_positive, shared_keys)) => (is_positive, shared_keys.x_i, shared_keys.y),
+        Err(e) => {
+            error!("Error: {:?}, state: {:?}", e, prev_state);
+            return prev_state.clone();
+        }
+    };
+
+    match prev_state {
+        NodeState::Master { .. } => Master::from_node_state(prev_state.clone())
+            .shared_block_secrets(new_shared_block_secrets)
+            .block_shared_keys(Some(block_shared_keys))
+            .build(),
+        NodeState::Member { .. } => Member::from_node_state(prev_state.clone())
+            .shared_block_secrets(new_shared_block_secrets)
+            .block_shared_keys(Some(block_shared_keys))
+            .build(),
+        _ => prev_state.clone(),
+    }
+}
+
+/// Store received vss
```

Comment:
This comment is not needed because it is the same as the name of the function.

Reply:
I will remove it.

Reply:
done!